docs/src/components/banner/AppFrameBanner.tsx

```diff
@@ -5,9 +5,15 @@ import FEATURE_TOGGLE from 'docs/src/featureToggle';
 import { alpha } from '@mui/material/styles';
 
 export default function AppFrameBanner() {
+
```

Comment:
The changes in this file are left to preview the alternative to replacing the hiring ad.